3_visualize/src/plot_site_data.R

```diff
@@ -7,6 +7,6 @@ plot_site_data <- function(out_file, site_data, parameter) {
     geom_line() +
     geom_point(data=filter(site_data, !(Quality %in% c('A','P'))), size=0.1) +
     ylab(dataRetrieval::parameterCdFile %>% filter(parameter_cd == parameter) %>% pull(parameter_nm)) +
-    ggtitle(site_data$Site[1])
+    ggtitle(site_data$State[1], site_data$Site[1])
```

Comment:
Cool, so you're probably getting the Site as a subtitle now (and the state abbreviation as the main title)? another way to do this that would give you more flexibility would be to use `sprintf()` to construct a string to pass as the first argument to `ggtitle`, e.g.,
```r
ggtitle(sprintf('%s-%s', site_data$State[1], site_data$Site[1]))
```
I'm sure you'll get lots more opportunities to create and refine ggplot titles in upcoming project work, too.